global.json

```diff
@@ -1,3 +1,6 @@
 {
-    "projects": [ "src", "test", "samples" ]
+    "projects": [ "src", "test", "samples" ],
+    "sdk": {
+        "version": "1.0.0-preview1-002702"

```

Comment:
> That corresponds to the version that you get of the tooling etc when you install the bits from https://www.microsoft.com/net/core

Huh. I literally just installed that 2 minutes ago and get

> $ dotnet --version
> 1.0.0-rc2-002392

Think I might have some old :hankey: lying around. The uninstall probably didn't work.